docs/app/Examples/elements/Label/Variations/LabelExampleSize.js

```diff
@@ -0,0 +1,19 @@
+import _ from 'lodash'
+import React from 'react'
+import { Label } from 'stardust'
+
+const LabelExampleSize = () => {
+  const sizes = ['mini', 'tiny', 'small', 'medium', 'large', 'big', 'huge', 'massive']
```

Comment:
More data here, let's move it up top. There are several other examples with data to update so I won't comment on all of them.